src/TConsole.h

```diff
@@ -325,10 +323,6 @@ public slots:
 
     ConsoleType mType = UnknownType;
     QSize mOldSize;
-    bool mAutoWrap = true;
-    SearchOptions mSearchOptions = SearchOptionNone;
-    QAction* mpAction_searchOptions = nullptr;
-    QIcon mIcon_searchOptions;
```

Comment:
These are associated with the "making the main console search have controllable case sensitivity" i.e. #6336 which shouldn't be being whipped out like this...

Reply:
Thanks, I'll try adding this and the below back in and pushing it up in a bit